geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLGeometricConstructorFunctions.scala

```diff
@@ -11,59 +11,22 @@ package org.apache.spark.sql
 import com.vividsolutions.jts.geom._
 import org.geotools.geometry.jts.JTS
 import org.locationtech.geomesa.utils.geohash.{BoundingBox, GeoHash}
-import org.locationtech.geomesa.utils.text.{WKBUtils, WKTUtils}
-import org.apache.spark.sql.SQLFunctionHelper.nullableUDF
+import org.locationtech.geomesa.spark.SQLFunctionHelper.nullableUDF
 
 object SQLGeometricConstructorFunctions {
 
   val ST_GeomFromGeoHash: (String, Int) => Geometry = nullableUDF((hash, prec) => GeoHash(hash, prec).geom)
-  val ST_GeomFromWKT: String => Geometry = nullableUDF(text => WKTUtils.read(text))
-  val ST_GeomFromWKB: Array[Byte] => Geometry = nullableUDF(array => WKBUtils.read(array))
-  val ST_LineFromText: String => LineString = nullableUDF(text => WKTUtils.read(text).asInstanceOf[LineString])
   val ST_MakeBox2D: (Point, Point) => Geometry = nullableUDF((lowerLeft, upperRight) =>
```

Comment:
do we not want to move these functions as well?

Reply:
I have! https://github.com/locationtech/geomesa/pull/1817/files/0edd96224b5a8bdecab7043816027da2fe166d14#diff-fb38b8272642f13cb7d7218f4f4a7054R19

Reply:
oh, should we remove them from here then?

Reply:
looks like `makebbox` and `makebox2d` are duplicates here? can we remove them?